Packetery/Checkout/etc/db_schema.xml

```diff
@@ -30,4 +32,94 @@
             <column name="id"/>
         </constraint>
     </table>
+
+    <table name="packetery_order" resource="default" engine="innodb" comment="Packetery Order Table">
+        <column xsi:type="int" name="id" padding="10" unsigned="true" nullable="false" identity="true" comment="Entity ID"/>
+        <column xsi:type="smallint" name="store_id" nullable="false" unsigned="true" comment="Magento Store ID"/>
+        <column xsi:type="varchar" name="order_number" nullable="false" length="128" comment="Order Number"/>
+        <column xsi:type="varchar" name="recipient_firstname" nullable="false" length="128" comment="Recipient Firstname"/>
+        <column xsi:type="varchar" name="recipient_lastname" nullable="false" length="128" comment="Recipient Lastname"/>
+        <column xsi:type="varchar" name="recipient_company" length="128" comment="Recipient Company"/>
+        <column xsi:type="varchar" name="recipient_email" length="128" comment="Recipient Email"/>
+        <column xsi:type="varchar" name="recipient_phone" length="32" comment="Recipient Phone"/>
+        <column xsi:type="decimal" name="cod" scale="4" precision="20" comment="COD"/>
+        <column xsi:type="varchar" name="currency" length="8" comment="Currency"/>
+        <column xsi:type="decimal" name="value" scale="4" precision="20" nullable="false" comment="Value"/>
+        <column xsi:type="decimal" name="weight" scale="4" precision="12" nullable="false" comment="Weight"/>
+        <column xsi:type="varchar" name="point_id" length="32" nullable="false" comment="Point ID"/>
+        <column xsi:type="varchar" name="point_name" length="1024" comment="Point Name"/>
+        <column xsi:type="boolean" name="is_carrier" nullable="false" default="0" comment="Is Point_id ID of external carrier?"/>
+        <column xsi:type="varchar" name="carrier_pickup_point" length="40" comment="External carrier pickup point ID"/>
+        <column xsi:type="varchar" name="sender_label" length="64" comment="Sender Label"/>
+        <column xsi:type="boolean" name="adult_content" comment="Adult Content"/>
+        <column xsi:type="date" name="delayed_delivery" comment="Delayed Delivery"/>
+        <column xsi:type="boolean" name="address_validated" nullable="false" default="0" comment="Is recipient address validated?"/>
```

Comment:
v původním installSchema bylo možné nastavit pořadí sloupců pomocí např. 'after' => 'delayed_delivery' . Jde to i zde ?

Reply:
V declarative schema to nyni mozne neni. Jediny zpusob, co jsem koukal, tak je to, ze se bude stale pouzivat upgradeSchema na pridani novych sloupcu, coz mi prijde ale takovy hybridni reseni a moc se mi to nelibi, tim ze jsme se dohodli, ze budeme pouzivat db_schema exkluzivne. Jak by sis pral toto vyresit?

Reply:
pokud parametr after zde použít nejde a pořadí se definuje pořadím sloupce jak je zapsaný v db_schema.xml, nebudeme to tedy řešit.  Použití UpgradeSchema se mi také nelíbí.